src/spring/azext_spring/_marketplace.py

```diff
@@ -14,8 +14,8 @@
 def _spring_list_marketplace_plan(cmd, client):
     # return get_mgmt_service_client(cli_ctx, AppPlatformManagementClient_20220501preview)
     from azure.cli.core.commands.client_factory import get_mgmt_service_client
-    from .vendored_sdks.marketplace.v2022_11_01_preview import MarketplaceRPService
-    from .vendored_sdks.marketplace.v2022_11_01_preview.models import Offer
+    from .vendored_sdks.marketplace.v2023_01_01_preview import MarketplaceRPService
+    from .vendored_sdks.marketplace.v2023_01_01_preview.models import Offer
```

Comment:
I have a small suggestion: if the SDK path we import does not contain the api-version part, then when upgrading the SDK version in the future, we only need to replace the SDK folder, not all the import paths.

Reply:
Very very good idea. As you can see, there are multi api version in our CLI extension, we plan to only use the latest API version in the future. There are 2 steps here.
1. Make sure we only use 1 API version in the code
2. Remove unused the API versions and remove the version info from the SDK file path.

We will create another PR to do the step 2.

Reply:
OK, sounds good!